examples/apps/generic_client/generic_client.py

```diff
@@ -190,6 +190,19 @@ def _lookup_first_worker(worker_registry, jrpc_req_id):
 
 
 def _do_worker_verification(worker_obj):
+    encryption_key_signature = worker_obj.encryption_key_signature
+    encryption_key = worker_obj.encryption_key
+    verify_key = worker_obj.verification_key
+
+    # Verify worker encryption key signature using worker's verification key
+    sig_obj = signature.ClientSignature()
```

Comment:
Can we create sig_obj only if encryption_key_signature is not None ? So you can move this line inside if statement below.

Reply:
Below statements can also be moved inside if block

```
encryption_key = worker_obj.encryption_key
verify_key = worker_obj.verification_key
```

Reply:
Done